requirements/extras.txt

```diff
@@ -0,0 +1,3 @@
+phoneme_guesser~=0.1
```

Comment:
This is duped from the main requirements

Reply:
removed from requirements and extras